molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/VcfValidator.java

```diff
@@ -90,6 +91,7 @@ public String validateVCF(File vcfFile)
 			bufferedWriter.close();
 
 			scanner.close();
+			errorScanner.close();
```

Comment:
Put close calls in finally block such that streams are closed in case an exception occurs.